build-system/tasks/helpers.js

```diff
@@ -350,7 +350,7 @@ function handleBundleError(err, continueOnError, destFilename) {
     // Drop the node_modules call stack, which begins with '    at'.
     message = err.stack.replace(/    at[^]*/, '').trim();
   }
-  log(red('ERROR:'), message, '\n');
+  log(red('ERROR:'), message.toString(), '\n');
```

Comment:
I think a better fix might be to change line 348 to `let message = err.toString()`. Then, `message` will always be a `string` and we can get rid of the type annotation on line 347.

Reply:
That does make sense